packages/next/pages/_document.js

```diff
@@ -187,7 +187,7 @@ export class NextScript extends Component {
   static getInlineScriptSource (documentProps) {
     const { __NEXT_DATA__ } = documentProps
     const { page } = __NEXT_DATA__
-    return `__NEXT_DATA__ = ${htmlescape(__NEXT_DATA__)};__NEXT_LOADED_PAGES__=[];__NEXT_REGISTER_PAGE=function(r,f){__NEXT_LOADED_PAGES__.push([r, f])};`
+    return `__NEXT_DATA__ = ${htmlescape(__NEXT_DATA__)};`
```

Comment:
Let’s remove the spaces around the `=`?

Reply:
We can keep it for now, this is going to be removed in my next PR 👍

Reply:
https://github.com/zeit/next.js/pull/5584